src/Compilers/CSharp/Portable/Symbols/TypedConstantExtensions.cs

```diff
@@ -2,25 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
+#nullable enable
+
 using System.Diagnostics;
 using System.Linq;
 using System.Text;
-using Microsoft.CodeAnalysis.Collections;
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.PooledObjects;
-using Microsoft.CodeAnalysis.Text;
-using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
     public static class TypedConstantExtensions
     {
         /// <summary>
         /// Returns the System.String that represents the current TypedConstant.
+        /// For testing and debugging only
```

Comment:
>For testing and debugging only [](start = 12, length = 30)

This method is part of the public API so I don't think we can state how this method is used.

Reply:
Thanks

---
In reply to: [443063564](https://github.com/dotnet/roslyn/pull/44266#discussion_r443063564) [](ancestors = 443063564)